docset/winserver2025-ps/smbshare/Reset-SmbServerConfiguration.md

```diff
@@ -143,6 +142,74 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuditClientCertificateAccess
+
+Enables auditing of the clients certificate access attempts on the server. Events are logged when a
```

Comment:
standby on this one, checking on exact behavior

Reply:
Also as a RESET value, all of these new ones are just resetting to default values. But these help entries seem to be for the SET commands, where values are expected - a problem in the MD